pkg/engine/plugins.go

```diff
@@ -163,10 +165,14 @@ func installPlugin(plugin workspace.PluginInfo) error {
 
 	logging.V(preparePluginVerboseLog).Infof(
 		"installPlugin(%s, %s): initiating download", plugin.Name, plugin.Version)
-	stream, _, err := plugin.Download()
+	stream, size, err := plugin.Download()
 	if err != nil {
 		return err
 	}
+
+	fmt.Printf("[%s plugin %s-%s] installing\n", plugin.Kind, plugin.Name, plugin.Version)
```

Comment:
Is this `Printf` intentional?

Reply:
Yes, to show the name of the plugin being installed (same as during `pulumi plugin install`)

Reply:
What does this end up looking like in the CLI? Can you provide a demo (e.g. via `asciinema`)?

Reply:
That tool doesn't seem to support Windows, but here is the output:

```
$pulumi preview
Previewing update (dev):
[resource plugin azure-1.0.0-beta.1] installing
Downloading plugin: 27.95 MiB / 39.31 MiB [=================>------]  71.10% 18s
```
and when finished
```
$pulumi preview
Previewing update (dev):
[resource plugin azure-1.0.0-beta.1] installing
Downloading plugin: 39.31 MiB / 39.31 MiB [========================] 100.00% 57s
Moving plugin... done.

     Type                                           Name     Plan                                                                            +   pulumi:pulumi:Stack                            aks-dev  create
```

Reply:
Thanks!